packages/@react-aria/list/src/useListItem.ts

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {focusSafely, getFocusableTreeWalker} from '@react-aria/focus';
+import {getRowId, listMap} from './utils';
+import {HTMLAttributes, KeyboardEvent as ReactKeyboardEvent, RefObject} from 'react';
+import {isFocusVisible} from '@react-aria/interactions';
+import type {ListState} from '@react-stately/list';
+import {mergeProps} from '@react-aria/utils';
+import {Node as RSNode} from '@react-types/shared';
+import {useLocale} from '@react-aria/i18n';
+import {useSelectableItem} from '@react-aria/selection';
+
+export interface AriaListItemOptions {
+  /** An object representing the list item. Contains all the relevant information that makes up the list row. */
+  node: RSNode<unknown>,
+  /** Whether the list row is contained in a virtual scroller. */
+  isVirtualized?: boolean,
+  /** Whether selection should occur on press up instead of press down. */
+  shouldSelectOnPressUp?: boolean,
+  /** Whether the list item is disabled. */
+  isDisabled?: boolean
+}
+
+export interface ListItemAria {
+  /** Props for the list row element. */
+  rowProps: HTMLAttributes<HTMLElement>,
+  /** Props for the grid cell element within the list row. */
+  gridCellProps: HTMLAttributes<HTMLElement>,
+  /** Whether the row is currently pressed. */
+  isPressed: boolean
```

Comment:
Added isPressed to the stuff that is returned from the hook to take advantage of the usePress call from useSelectableItem.